cli/src/lib/zksync-era-state.ts

```diff
@@ -23,33 +15,39 @@ import {
   SystemContractList,
   type SystemContractProvider,
 } from "./system-contract-providers";
-import {
-  callDataSchema,
-  type FacetCut,
-  l2UpgradeSchema,
-  upgradeCallDataSchema,
-} from "../schema/rpc";
 import { z } from "zod";
+import { RecordStorageSnapshot } from "./storage/record-storage-snapshot";
+import {
+  AddressExtractor,
+  BigNumberExtractor,
+  BlobExtractor,
+  FacetsToSelectorsVisitor,
+  ListOfAddressesExtractor,
+} from "./reports/extractors";
+import type { ContractField } from "./storage/contractField";
+import type { StorageSnapshot } from "./storage/storage-snapshot";
+import type { StorageVisitor } from "./reports/storage-visitor";
+import { type CallTrace, l2UpgradeSchema, upgradeCallDataSchema } from "../schema/rpc";
 
 export type L2ContractData = {
   address: Hex;
   bytecodeHash: Hex;
   name: string;
 };
 
-export enum PubdataPricingMode {
```

Comment:
might as well delete these, we can rely on source control if need it back again

Reply:
Totally